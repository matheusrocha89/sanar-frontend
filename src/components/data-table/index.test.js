import React from 'react';
import renderer from 'react-test-renderer';

import DataTable from './';


describe('DataTable', () => {
  it('renders correctly', () => {
    const data = [
      { id: 1, profissao: 'test', instituto: 'test' },
      { id: 2, 'profissao': 'test', instituto: 'test' },
    ];
    const tree = renderer.create(<DataTable data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
