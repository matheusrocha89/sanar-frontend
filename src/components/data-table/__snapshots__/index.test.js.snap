// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable renders correctly 1`] = `
<div
  className="table-responsive"
>
  <table
    className="table table-bordered table-condensed table-hover"
    stripped="true"
  >
    <thead>
      <tr>
        <th>
          ID
        </th>
        <th>
          Profiss√£o
        </th>
        <th>
          Instituto
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          1
        </td>
        <td>
          test
        </td>
        <td>
          test
        </td>
      </tr>
      <tr>
        <td>
          2
        </td>
        <td>
          test
        </td>
        <td>
          test
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
